src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -50,9 +53,34 @@ const DefElemOptionFormat create_database_option_formats[] = {
 	{ "tablespace", " TABLESPACE %s", OPTION_FORMAT_STRING },
 	{ "allow_connections", " ALLOW_CONNECTIONS %s", OPTION_FORMAT_BOOLEAN },
 	{ "connection_limit", " CONNECTION LIMIT %d", OPTION_FORMAT_INTEGER },
-	{ "is_template", " IS_TEMPLATE %s", OPTION_FORMAT_BOOLEAN }
+	{ "is_template", " IS_TEMPLATE %s", OPTION_FORMAT_BOOLEAN },
+	{ "oid", " OID %d", OPTION_FORMAT_OBJECT_ID }
 };
 
+
+const DefElemOptionFormat alterDatabaseOptionFormats[] = {
+	{ "is_template", " IS_TEMPLATE %s", OPTION_FORMAT_BOOLEAN },
+	{ "allow_connections", " ALLOW_CONNECTIONS %s", OPTION_FORMAT_BOOLEAN },
+	{ "connection_limit", " CONNECTION LIMIT %d", OPTION_FORMAT_INTEGER },
+};
+
+
+/*
+ * DeparseAlterDatabaseOwnerStmt
+ *      Deparse an AlterDatabaseOwnerStmt node
+ *
+ * This function is responsible for producing a string representation of an
+ * AlterDatabaseOwnerStmt node, which represents an ALTER DATABASE statement
+ * that changes the owner of a database. The output string includes the ALTER
+ * DATABASE keyword, the name of the database being altered, and the new owner
+ * of the database.
+ *
+ * Parameters:
+ *  - node: a pointer to the AlterDatabaseOwnerStmt node to be deparsed
+ *
+ * Returns:
+ *  - a string representation of the ALTER DATABASE statement
+ */
```

Comment:
We don't really add comments to Deparse functions unless there is something that's worth to explain (e.g., explaining Basic options in AppendBasicAlterDatabaseOptions is nice).

```suggestion
```